pgmpy/utils/utils.py

```diff
@@ -98,25 +99,63 @@ def get_example_model(model):
         "pathfinder": "utils/example_models/pathfinder.bif.gz",
         "pigs": "utils/example_models/pigs.bif.gz",
         "munin": "utils/example_models/munin.bif.gz",
-        "ecoli70": "",
+        "ecoli70": "utils/example_models/ecoli70.json",
         "magic-niab": "",
         "magic-irri": "",
         "arth150": "",
         "sangiovese": "",
         "mehra": "",
     }
 
-    if model not in filenames.keys():
-        raise ValueError("dataset should be one of the options")
-    if filenames[model] == "":
-        raise NotImplementedError("The specified dataset isn't available.")
+    if model not in filenames:
+        raise ValueError(f"Unknown model name: {model}")
 
     path = filenames[model]
-    ref = files("pgmpy") / path
-    with gzip.open(ref) as f:
-        content = f.read()
-    reader = BIFReader(string=content.decode("utf-8"), n_jobs=1)
-    return reader.get_model()
+
+    # Handle BIF files (existing logic)
+    if path.endswith(".bif.gz"):
```

Comment:
@B3CODER For readability I think it would be better that we define variables with all the possible categorical and continuous models. Then, check whether the model is categorical or continuous. Something like:
```
cat_models = {'asia', 'cancer', 'earthquake', .... }
cont_models = {'ecoli70', 'magic-niab', ... }

if model in cat_models:
      # Read categorical model
elif model in cont_models:
     # Read continuous model
else:
     raise error
```